### ISSUE TITLE
pip install with python 3.8

### ISSUE THREAD

**Issue by mglisse** [PythonOT/POT#136]
**Describe the bug**
`python3.8 -m pip install POT`
fails on debian testing x86_64.

The error messages look like
<pre>
  ot/lp/emd_wrap.cpp: In function ‘PyObject* __Pyx_PyCFunction_FastCall(PyObject*, PyObject**, Py_ssize_t)’:
  ot/lp/emd_wrap.cpp:6660:69: error: too many arguments to function
       return (*((__Pyx_PyCFunctionFast)meth)) (self, args, nargs, NULL);
                                                                       ^
</pre>

I suspect (did not check) this means that a .cpp file was generated for the pip package with an old cython version incompatible with newer python. Building from a git checkout works just fine here (I have cython 0.29.14).

**Desktop (please complete the following information):**
 - OS: Linux Debian testing x86_64
 - Python version: 3.8
- How was POT installed: pip

Output of the following code snippet:
```python
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("NumPy", numpy.__version__)
import scipy; print("SciPy", scipy.__version__)
import ot; print("POT", ot.__version__)
```
<pre>
>>> import platform; print(platform.platform())
Linux-5.4.0-4-amd64-x86_64-with-glibc2.29
>>> import sys; print("Python", sys.version)
Python 3.8.2 (default, Feb 25 2020, 13:04:52) 
[GCC 9.2.1 20200220]
>>> import numpy; print("NumPy", numpy.__version__)
NumPy 1.17.4
>>> import scipy; print("SciPy", scipy.__version__)
SciPy 1.3.3
>>> import ot; print("POT", ot.__version__)
/usr/lib/python3/dist-packages/cvxopt/__init__.py:136: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if val is 0: val = None
/usr/lib/python3/dist-packages/cvxopt/printing.py:43: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if width*height is 0: return ""
/usr/lib/python3/dist-packages/cvxopt/printing.py:44: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(X) is 0: return ""
/usr/lib/python3/dist-packages/cvxopt/printing.py:101: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if width*height is 0: return ""
/usr/lib/python3/dist-packages/cvxopt/coneprog.py:4043: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if status is 'optimal' or dcost <= 0.0:
/usr/lib/python3/dist-packages/cvxopt/coneprog.py:4061: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if status is 'optimal' or pcost >= 0.0:
POT 0.6.0
</pre>
The POT 0.6.0 that is installed is the one I built from a git checkout.

**Comment by mglisse** [comment]
Fixed with 0.7.0.

**Comment by rflamary** [comment]
Thank you for the information I was  hoping it would be fixed but now i'm sure ;)
